app/ExposureNotificationContext.tsx

```diff
@@ -14,9 +42,43 @@ interface ExposureNotificationsState {
 
 const initialStatus: ENAuthorizationStatus = 'notAuthorized';
 
+const now: Posix = Date.now();
+
+const daysAgo = [...Array(21)].map((_v, idx: number) => {
+  return 20 - idx;
+});
+
+const initialExposureHistory: ExposureHistory = daysAgo.map(
```

Comment:
@mattThousand, (cc: rebecca) this is all fake data for what it looks like we will need to populate the exposure notification history screen.

Reply:
lets find a time to discuss it would be good to get the schema well thought out, this is based off what makes sense for the requested views, not sure if its the best structure for what is going on on the native side.